base/applications/regedit/treeview.c

```diff
@@ -625,6 +625,161 @@ BOOL CreateNewKey(HWND hwndTV, HTREEITEM hItem)
     return bSuccess;
 }
 
+static VOID
```

Comment:
Is there a reason for moving this function to another file ?

Reply:
Yes. Since it's a treeview-specific function, it's right place is in the treeview.c file, not the main GUI file.
That's how it's organized for listview specific function (listview.c) and other treeview specific functions.

Reply:
As already said in mattermost, that makes it easy harder to review so most people will just ignore this

Reply:
Yes, but for the part that has been moved and not modified, the need for in-depth review is lower (built is OK and functional test too, to repvent regression).

UpdateAddress could be "disregarded" during review and focus be exclusively on TreeWndNotifyProc and the "small" changes in orther parts (which are the bug-fixing part).

I hope this simplifies the review process.

Reply:
@learn-more : Would you prefer that I focus exclusively on the bug fixing part and "revert" the code maintenability part to its original state (and postpone this enhancement to a future PR) ?

Reply:
Reverted back. This un-modified function is now back to its original place (but no longer static, in order to be called from Treeview.c)